gear/gear/auth.py

```diff
@@ -119,6 +120,19 @@ async def _fetch_userdata_from_auth_service(session_id_and_session: Tuple[str, h
             raise web.HTTPInternalServerError() from e
 
 
+class TrustedSingleTenantAuthenticator(Authenticator):
+    async def _fetch_userdata(self, _: web.Request) -> Optional[UserData]:
+        return cast(
+            UserData,
+            {
+                'is_developer': True,
+                'username': 'test-dev',
```

Comment:
We should maybe use `user`?

Reply:
Ya, we might want to add a small script after the migrations that, say, creates a billing project and adds them to it though.